intro.qmd

```diff
@@ -320,11 +320,23 @@ By default, this compact display of mixed-effects model fits is used in [Jupyter
 To obtain more information we can `print` the model, as in
 
 ```{julia}
-println(dsm01)
+print(dsm01)
 ```
 
 The call to `fit` constructs a `LinearMixedModel` object, evaluates the [maximum likelihood](https://en.wikipedia.org/wiki/Maximum_likelihood_estimation) parameter estimates, assigns the results to the name `dsm01`, and displays a summary of the fitted model.
 
+:::{.callout-note collapse="true"}
+### The named argument, progress=false
+
+The optional, named argument, `progress=false`, suppresses the creation and display of a progress meter for the iterations.
+Such displays are useful in interactive sessions to monitor the progress of the iterations but not in a static display for a book.
+The comma used to separate argument values can be replaced by a semi-colon when it separates positional arguments, which must precede named arguments, from the first named argument.
```

Comment:
I don't have a better phrasing suggestion at the moment, but I had to read this twice to understand it and I know the behavior in question.

Reply:
I modified the text to what I hope is a more comprehensible passage.

Reply:
Yes, much better!